locales/fr/translation.json

```diff
@@ -0,0 +1,25 @@
+{
+    "Federated Image Sharing": "Partage fédéré d'images",
+    "Powered by": "Propulsé par",
+    "and the": "et le",
+    "fediverse": "fediverse",
+    "Clean UI": "Interface propre",
+    "Inspired by Modern Design": "Inspiré par un design moderne",
+    "Familiar Features": "Fonctionnalité connues",
```

Comment:
«Fonctionnalités familières»

Reply:
“Familiar Features” is said in the sense that features in PixelFed are ± the same of Instagram, so « connues » is better.